src/api/api.types.tsx

```diff
@@ -250,13 +250,14 @@ export interface ImagePost {
 }
 
 export interface APIImage
-  extends Required<Omit<ImagePost, 'upload_file'>>,
+  extends Required<Omit<ImagePost, 'upload_file' | 'entity_id'>>,
     CreatedModifiedMixin {
   id: string;
   primary: boolean;
   thumbnail_base64: string;
 }
 
 export interface APIImageWithURL extends APIImage {
-  url: string;
+  view_url: string;
+  download_url: string;
```

Comment:
add a comment on what is different about these two urls and why is it being used in this way